RecoTracker/IterativeTracking/python/iterativeTk_cff.py

```diff
@@ -43,6 +43,25 @@
     InitialStepPreSplitting +
     InitialStep +
     HighPtTripletStep +
+    DetachedQuadStep +
```

Comment:
`Detached*Step` is after `LowPt*Step` in `trackingPhase1PU70` because of historical reasons (that was their order in Run1 tracking). In Run2 tracking `Detached*Step` is before `LowPt*Step` at least because that way is faster (I don't remember if it has any notable impact on physics performance), and thus that is the order in `trackingPhase1`.